scenes/main_menu/main_menu_level.gd

```diff
@@ -2,20 +2,29 @@
 extends "../scene_base.gd"
 
 func _update_demo_positions() -> void:
-	var count = $Demos.get_child_count()
+	var count = 0#$Demos.get_child_count()
+	var vis_chd := []
+	for _tp in $Demos.get_children():
+		_tp.active=_tp.visible
+		if _tp.visible:
+			count+=1
+			vis_chd.append(_tp)
 	if count > 1:
 		var angle = 2.0 * PI / count
 		for i in count:
 			var t = Transform3D()
 			t.origin = Vector3(0.0, 0.0, -10.0)
 			t = t.rotated(Vector3.UP, angle * i)
 
-			$Demos.get_child(i).transform = t
+			vis_chd[i].transform = t
 
 
 func _ready():
 	super._ready()
 	_update_demo_positions()
+	
+	for _c in $Demos.get_children():
```

Comment:
Same here, just rename `_c` to `child` or keep it consistent and all it `teleporter`

Reply:
To be fair it was _c in begining, but later I added `teleport.active`

Reply:
Also I think I gona go with world `teleporter` cause I think it will be faster to understand when somebody copies repo/downloads demos